### ISSUE TITLE
error for showing error message

### ISSUE THREAD

**Issue by kibitan** [kibitan/masking#73]
* with no `masking.yml`
 * without `--complete-insert` option (invalid format)

```
$ ruby -v
ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [arm64-darwin21]
$ gem install masking
Fetching masking-1.1.1.gem
Successfully installed masking-1.1.1
Parsing documentation for masking-1.1.1
Installing ri documentation for masking-1.1.1
Done installing documentation for masking after 0 seconds
1 gem installed
$ which masking
/Users/chikahiro.tokoro/.asdf/shims/masking
$ masking -v
1.1.1
$ masking -c aa
/Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli/error_message.rb:25:in `read': No such file or directory @ rb_sysopen - lib/masking/cli/error_messages.yml (Errno::ENOENT)
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli/error_message.rb:25:in `read'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli/error_message.rb:25:in `error_messages'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli/error_message.rb:30:in `error_message'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli/error_message.rb:15:in `message'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli.rb:19:in `rescue in run'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli.rb:15:in `run'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/exe/masking:10:in `<top (required)>'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/bin/masking:25:in `load'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/bin/masking:25:in `<main>'
/Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/config/target_columns.rb:17:in `initialize': Masking::Error::ConfigFileDoesNotExist (Masking::Error::ConfigFileDoesNotExist)
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/config.rb:27:in `new'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/config.rb:27:in `target_columns_file_path='
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli.rb:39:in `block (2 levels) in define_config_option'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/config.rb:13:in `configure'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli.rb:38:in `block in define_config_option'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/optparse.rb:1614:in `block in parse_in_order'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/optparse.rb:1559:in `catch'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/optparse.rb:1559:in `parse_in_order'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/optparse.rb:1553:in `order!'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/optparse.rb:1659:in `permute!'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/optparse.rb:1684:in `parse!'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/3.1.0/optparse.rb:1673:in `parse'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/lib/masking/cli.rb:16:in `run'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/masking-1.1.1/exe/masking:10:in `<top (required)>'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/bin/masking:25:in `load'
	from /Users/chikahiro.tokoro/.asdf/installs/ruby/3.1.2/bin/masking:25:in `<main>'
```

**Comment by kibitan** [comment]
i don't know how to reproduce.  my machine & #75 can't reproduce it (yet)

**Comment by kibitan** [comment]
it can be reproducible to use docker container

```
$ mysqldump -uroot --all-databases | docker run -i ghcr.io/kibitan/masking
/usr/local/bundle/gems/masking-1.1.1/lib/masking/cli/error_message.rb:25:in `read': No such file or directory @ rb_sysopen - lib/masking/cli/error_messages.yml (Errno::ENOENT)
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/cli/error_message.rb:25:in `read'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/cli/error_message.rb:25:in `error_messages'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/cli/error_message.rb:30:in `error_message'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/cli/error_message.rb:15:in `message'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/cli.rb:19:in `rescue in run'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/cli.rb:15:in `run'
	from /usr/local/bundle/gems/masking-1.1.1/exe/masking:10:in `<top (required)>'
	from /usr/local/bundle/bin/masking:25:in `load'
	from /usr/local/bundle/bin/masking:25:in `<main>'
/usr/local/bundle/gems/masking-1.1.1/lib/masking/config/target_columns.rb:17:in `initialize': Masking::Error::ConfigFileDoesNotExist (Masking::Error::ConfigFileDoesNotExist)
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/config.rb:31:in `new'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/config.rb:31:in `target_columns'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/data_mask_processor.rb:11:in `initialize'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/sql_dump_line.rb:13:in `new'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/sql_dump_line.rb:13:in `mask'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking.rb:23:in `block in run'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking.rb:22:in `each_line'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking.rb:22:in `run'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking.rb:10:in `run'
	from /usr/local/bundle/gems/masking-1.1.1/lib/masking/cli.rb:17:in `run'
	from /usr/local/bundle/gems/masking-1.1.1/exe/masking:10:in `<top (required)>'
	from /usr/local/bundle/bin/masking:25:in `load'
	from /usr/local/bundle/bin/masking:25:in `<main>'
-- MySQL dump 10.13  Distrib 5.7.41, for osx10.17 (x86_64)
--
-- Host: localhost    Database:
-- ------------------------------------------------------
-- Server version	5.7.41

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
...
```

**Comment by kibitan** [comment]
fixed by #75, with checking `1.1.2-alpha-pre` release

```
$ docker run masking -v
1.1.2-alpha

$ mysqldump -uroot --all-databases | docker run -i masking
ERROR: config file (masking.yml) does not exist
-- MySQL dump 10.13  Distrib 5.7.41, for osx10.17 (x86_64)
--
-- Host: localhost    Database:
-- ------------------------------------------------------
-- Server version	5.7.41

...
```

```
$ mysqldump -uroot --all-databases | docker run -i -v $(pwd)/acceptance/masking.yml:/app/masking.yml ghcr.io/kibitan/masking
....
LOCK TABLES `db` WRITE;
/*!40000 ALTER TABLE `db` DISABLE KEYS */;
ERROR: cannot parse SQL dump file. you may forget to put `--complete-insert` option in mysqldump?
mysqldump: Got errno 32 on write
```
